src/main/java/mate/jdbc/service/impl/DriverServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao dao;
```

Comment:
```suggestion
    private DriverDao driverDao;
```
This service has only one class. But it happens that there can be several.